apps/backoffice-v2/src/pages/Entities/components/Cases/hooks/useCases/useCases.tsx

```diff
@@ -104,15 +152,16 @@ export const useCases = () => {
         if (!sortRef.current) break;
 
         sortRef.current.focus();
+        break;
       case 'f':
```

Comment:
Adding curly braces allows using `const`. Up to you if to leave in these changes.
```suggestion
      case 'f': {
```